components/places/examples/autocomplete.rs

```diff
@@ -262,16 +262,11 @@ mod autocomplete {
     #[derive(Debug, Clone)]
     struct ConnectionArgs {
         path: PathBuf,
-        encryption_key: Option<String>,
     }
 
     impl ConnectionArgs {
         pub fn connect(&self) -> Result<places::PlacesDb> {
-            let key = match &self.encryption_key {
-                Some(k) => Some(k.as_str()),
-                _ => None,
-            };
-            let api = places::PlacesApi::new(self.path.to_str().unwrap(), key)?;
+            let api = places::PlacesApi::new(self.path.to_str().unwrap())?;
```

Comment:
I know this patch didn't add the unwrap() here, and it seems extremely unlikely to be an issue in the way the component if currently used, but we do have an `IllegalDatabasePath` error, so ISTM we might as well use it here?